### PR TITLE
Core/Custom: Fix Dream Vision & Eye of Kilrogg aggro and visibility.

### PR DESCRIPTION
**Pull request by DevRival**
Changes proposed: Fix Dream Vision & Eye of Kilrogg aggro and visibility

Target branch(es): 1.x

Tests performed: (Does it build, tested in-game, etc)
Tested in-game , works as intended.

Known issues and TODO list: No issue.

### CODE REVIEW

File: data/sql/updates/2016_08_14_00.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ALTER TABLE world_db_version CHANGE COLUMN 2016_08_10_01 2016_08_14_00 bit;
+
+-- Dream Vision make it trigger
+UPDATE `creature_template` SET `flags_extra`='128'WHERE (`entry`='7863');
```
Comment (ShinDarth) [inline data/sql/updates/2016_08_14_00.sql]: shouldn't be there a space before WHERE ?

**Comment by ShinDarth** [comment]
can you squash your commits?

https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Comment by Yehonal** [comment]
Shin it's not necessary, you can do via github directly

Il 14/Ago/2016 10:49, "Shin" <notifications@github.com> ha scritto:

**Comment by DevRival** [comment]
@Yehonal Thank you, I know, but I do not like github and soft, I'm a fan bitbucket.

**Comment by DevRival** [comment]
@Yehonal Thank you , I know, but I do not like github and soft , I'm a fan bitbucket.

**Pull request merged**